blog/app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ArticlesController < ApplicationController
+
+  http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
```

Comment:
While this was included in the tutorial, we won't be using this in class. Make sure you know what this does though!